src/Sulu/Bundle/PageBundle/Resources/config/forms/page_block_settings.xml

```diff
@@ -26,6 +26,129 @@
             <tag name="sulu.block_setting_icon" icon="su-hide" />
         </property>
 
+        <section name="schedules">
+            <properties>
+                <property name="schedules_enabled" type="checkbox">
+                    <params>
+                        <param name="description">
+                            <meta>
+                                <title>sulu_page.schedules_description</title>
+                            </meta>
+                        </param>
+                        <param name="icon" value="su-clock" />
+                        <param name="label">
+                            <meta>
+                                <title>sulu_page.schedules_label</title>
+                            </meta>
+                        </param>
+                        <param name="skin" value="heading" />
+                        <param name="type" value="toggler" />
+                    </params>
+
+                    <tag name="sulu.block_setting_icon" icon="su-clock" />
+                </property>
+
+                <block name="schedules" minOccurs="1" visibleCondition="schedules_enabled == true">
+                    <params>
+                        <param name="collapsable" value="false" />
+                        <param name="movable" value="false" />
```

Comment:
would still be nice to set a `add_button_text`

Reply:
Sorry, already implemented it but refactored that part afterwards :see_no_evil: